Resources/Prototypes/Entities/Mobs/Species/human.yml

```diff
@@ -128,6 +128,12 @@
       layer:
       - Opaque
       - MobImpassable
+  - type: AtmoExposable
+  - type: Temperature
+    fireDamageThreshold: 373
+    coldDamageThreshold: 173
+    specificHeat: 42
+    tempDamageCoefficient: 0.1
```

Comment:
Hm. I'm not sure if this is the way to go about this, maybe this should be in the domain of body system for humans.
In SS13, for example, human bodies can cool/heat itself slightly to resist high/low temperatures, and heat/cold damage thresholds are defined by normal body temperature + 50

Reply:
I left it as is and added some kind of thermal regulation to body system